packages/amplication-client/src/Plugins/hooks/usePlugins.ts

```diff
@@ -6,58 +6,102 @@ import {
 } from "../queries/pluginsQueries";
 import * as models from "../../models";
 import { keyBy } from "lodash";
-import { useContext, useMemo } from "react";
+import { useCallback, useContext, useMemo } from "react";
 import { AppContext } from "../../context/appContext";
 
 export type Plugin = {
   id: string;
   name: string;
   description: string;
-  logo: string;
+  repo: string;
+  npm: string;
+  icon: string;
+  github: string;
+  website: string
+  category: string;
+  type: string;
 };
 
+export type OnPluginDropped = (dragItem: models.PluginInstallation, hoverItem: models.PluginInstallation) => void;
+
 const PLUGINS: Plugin[] = [
   {
     id: "@amplication/plugin-auth-jwt",
     description:
       "A Passport strategy for authenticating with a JSON Web Token (JWT).",
-    logo: "logo",
+    icon: "",
     name: "Passport JWT Authentication",
+    repo: "test",
+    npm: "test",
+    github: "test",
+    website: "test",
+    category: "test",
+    type: "test",
   },
   {
     id: "@amplication/plugin-auth-basic",
     description:
       "A Passport strategy for authenticating using the standard basic HTTP scheme.",
-    logo: "logo",
+    icon: "",
     name: "Passport Basic Authentication",
+    repo: "test",
+    npm: "test",
+    github: "test",
+    website: "test",
+    category: "test",
+    type: "test",
   },
   {
     id: "@amplication/plugin-monorepo-nx",
     description:
       "Add the required configurations and files to use nx to manage your monorepo",
-    logo: "logo",
+    icon: "",
     name: "NX monorepo",
+    repo: "test",
+    npm: "test",
+    github: "test",
+    website: "test",
+    category: "test",
+    type: "test",
   },
   {
     id: "@amplication/plugin-db-postgres",
```

Comment:
It was agreed that the id property will hold the name of the plugin without the prefix and will be identical to the folder name that holds the plugin in the repo. 
So, we should do one of the below:
1. Send the value from "npm" to the id on the server 
2. Add another property to PluginInstallation to hold the value of "npm" in addition to the id